pkg/scaffold/cmd.go

```diff
@@ -48,15 +48,19 @@ import (
 
 	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
 	"github.com/operator-framework/operator-sdk/pkg/leader"
+	"github.com/operator-framework/operator-sdk/pkg/metrics"
 	"github.com/operator-framework/operator-sdk/pkg/ready"
 	sdkVersion "github.com/operator-framework/operator-sdk/version"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
 	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
 )
 
+// Change metricsAddress to serve metrics on different host and port.
+var metricsAddress string = ":8383"
```

Comment:
Do we envision users ever needing to specify anything for the host portion?

There's probably something I'm not thinking about, but following along with my question about the type of the `ExposeMetricsPort` parameter, could we make this an `int32` and use `fmt.Sprintf(":%d", metricsPort)` for the `MetricsBindAddress` in the manager options?

Reply:
>Do we envision users ever needing to specify anything for the host portion?

Yes, the user can pass the host part to controller-runtime, as in they might want to only have the server bind to `127.0.0.1` instead of all `0.0.0.0`, etc.

> but following along with my question about the type of the ExposeMetricsPort parameter

To answer this here as well: While it’s not necessarily needed to pass host to the `ExposeMetricsPort  to create the service file and expose the port there, it doesn’t hurt to encapsulate the retrieval of port in that function. WDYT? But can rename the function name there?

Reply:
> it doesn’t hurt to encapsulate the retrieval of port in that function. WDYT? But can rename the function name there?

My concern is mainly that users may think that `ExposeMetricsPort()` uses the host portion of the address and may be confused when changing it has no effect within that function.

Reply:
> My concern is mainly that users may think that ExposeMetricsPort() uses the host portion of the address and may be confused when changing it has no effect within that function.

 Fair concern! But the other option is to either:
1. remove the host part on the user side, but that gives us the same problem as we had with creating the Service object in the `main.go`. 
2. Another option is:
> use fmt.Sprintf(":%d", metricsPort) for the MetricsBindAddress in the manager 
I fear this would be missed by the user, and they would not know they can or have to change it? 

wdyt?

Reply:
We could also declare two variables in the header and still use fmt.Sprintf() to combine them for the MetricsBindAddress option.

```go
var (
    metricsHost = "0.0.0.0"
    metricsPort int32 = 8383
)

```